src/App.scss

```diff
@@ -73,84 +104,109 @@ body {
     margin-left: 50px;
 }
 
-.App__program-section {
-    margin-top: 1.5rem;
+.App__world-selector-container {
```

Comment:
This may change after the latest 286 gets merged in, but world-selector container need same amount of margin-right as the container for ProgramBlockEditor. Let's add the margin.

`margin-right: 0.5rem`

Reply:
Done.